directory.json

```diff
@@ -88,6 +88,7 @@
 	"MOZSL": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Sri%20Lanka/MozillaSriLanka-api.json",
 	"MTH": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Thailand/MozillaThailand-api.json",
 	"MUUG": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Mongolia/MongolianUnixUserGroup-api.json",
+	"Nastya":"https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Ukraine/Fixis-api.json",
```

Comment:
Hey, Shouldnt the Community name go up there instead of your name i.e. in my opinion it should be `"Fixis": "https://raw.githubusercontent.com/fossasia/fossasia-communities/master/Ukraine/Fixis-api.json"` ?